mods/cnc/chrome/lobby-mappreview.yaml

```diff
@@ -36,6 +30,14 @@ Container@LOBBY_MAP_PREVIEW:
 					Height: 25
 					Font: Tiny
 					Align: Center
+				LabelWithTooltip@MAP_TITLE:		# MAP_TITLE is displayed above MAP_TYPE, MAP_AUTHOR, but to be in front of them for tooltip it was moved here below
```

Comment:
In my test, the tooltip wasn't displayed immediately as expected, I had to move mouse in middle of label. I assume it is because those 3 labels are overlapping. When I changed height, it looked different/worse. The simplest solution I found was to move it after (to be above those 2 labels) so the tooltip is displayed properly.